### ISSUE TITLE
IDEA plugin: Ability to launch browsers directly from js-test-driver plugin pane

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#152]
```
PHPStorm/WebStorm has configuration options where paths to all browsers is 
specified.

It will be very useful if one can lanuch configured browsers directly from 
js-test-driver plugin's pane with capture URL opened. I.e. by double-clicking 
browser's icon (which is already displayed at plugin's pane)
```

Original issue reported on code.google.com by `olegelif...@gmail.com` on 25 Oct 2010 at 9:28

**Comment by GoogleCodeExporter** [comment]
```
I didn't realize Php/WebStorm already know the browser paths, but it makes 
sense that they have a preview function.

Great idea to capture browsers this way. Will try to get to it soon...
```

Original comment by `aeagle22206` on 21 Nov 2010 at 7:05
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
It didn't "knows" real picture, but there is a settings page for it (Settings 
-> Web Browsers)
```

Original comment by `olegelif...@gmail.com` on 26 Nov 2010 at 8:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 11 Apr 2011 at 7:33
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 117.65 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 27 Mar 2012 at 9:32
- Changed state: **Fixed**
